sdk/maps/maps-common/src/models/pollers.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CancelOnProgress, PollOperationState, PollerLike } from "@azure/core-lro";
+import { AbortSignalLike } from "@azure/abort-controller";
+
+/** Batch operation poller interface */
+export interface BatchPoller<TBatchResult>
+  extends PollerLike<PollOperationState<TBatchResult>, TBatchResult> {}
+
+/**
+ * Batch operation poller proxy
+ */
+export class BatchPollerProxy<TBatchResult, TInternalBatchResult>
+  implements BatchPoller<TBatchResult>
+{
+  constructor(
+    private internalPoller: PollerLike<
```

Comment:
Does your LROs support cancellation?

Reply:
The `internalPoller` only accepts the generated poller (e.g. [`beginFuzzySearchBatch`](https://github.com/andykao1213/azure-sdk-for-js/blob/andrewkao/maps-search/sdk/maps/maps-search/src/generated/operations/search.ts#L827)), which is a wrapper of `LroEngine`. But I see that the `cancelOperation` is deprecated. So should I remove this?

Reply:
@deyaaeldeen per the previous discussion, since map-search's lro doesn't support cancelation, I've hidden it in the commit 1de07d2.
I will open another PR before GA to replace the generated lro with `createHttpPoller` as you suggest.